__tests__/app.test.js

```diff
@@ -79,12 +79,92 @@ describe("GET: /api/articles/:article_id", () => {
 })
 
 describe("ALL: /api/:any_unknown_endpoint", () => {
-    it("Status 400: resposnds with a 400 bad request and an appropriate message when an invalid enfd point is provided.", () => {
+    it("Status 400: resposnds with a 400 bad request and an appropriate message when an invalid end point is provided.", () => {
         return request(app)
             .get('/api/load_of_rubbish123')
             .expect(400)
             .then((  {body} )=> {
                 expect(body.msg).toBe("Invalid input, no such end point exists");
             })
     })
+})
+
+describe("PATCH: /api/articles/:article_id", () => {
+    describe("HAPPY path", () => {
+        it("Status 200: responds with a 200 to indicate that the patch was succesful and responds with the updated article.", () => {
+            const incVotes = {
+                inc_votes : 100
+            };
+            return request(app)
+                .patch('/api/articles/2')
+                .send(incVotes)
+                .expect(200)
+                .then(( {body} ) => {
+                    const { article } = body
+                    expect(article).toBeInstanceOf(Object);
+
+                    expect(article).toEqual(
+                        expect.objectContaining({
+                            article_id: expect.any(Number),
```

Comment:
consider checking these against the actual values on the object, because this test should check against a specific article